torch/nn/functional.py

```diff
@@ -437,6 +437,12 @@ def relu(input, inplace=False):
     return _functions.thnn.Threshold.apply(input, 0, 0, inplace)
 
 
+def glu(input, dim=-1):
+    if dim < 0:
+        dim += input.dim()
```

Comment:
I don't think you can ignore checks in here, or we're going to give weird error message.
```python
input = torch.randn(2, 2)
glu(input, dim=-4)
```
will fail with an error that dim -2 is out of range (while -2 is in fact a valid value for dim).